misk-hibernate/src/main/kotlin/misk/hibernate/RealTransacter.kt

```diff
@@ -232,21 +234,45 @@ internal class RealTransacter private constructor(
           session.postCommit()
           result
         } catch (e: Throwable) {
+          var rethrow = e
```

Comment:
unrelated?

Reply:
looks like there's a few more unrelated file changes here.
If it's not strictly related to loading keys from S3, can we put those changes in a different PR?

Reply:
Hm this shouldn't be part of this PR, i assume this was an unintended rebase.